.github/workflows/ci.yaml

```diff
@@ -0,0 +1,68 @@
+# This workflow will install Python dependencies, run tests and lint with a matrix of OS and Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: [3.6, 3.7, 3.8, pypy3]
+        exclude:
+        - os: windows-latest
+          python-version: 3.6
+        - os: windows-latest
+          python-version: pypy3
+
+    steps:
```

Comment:
I think CI should reuse the `scripts/` as much as possible. Otherwise there's a lot of duplication and risk of discrepencies between CI and local testing.

If we need to add stuff to `scripts/install`/`scripts/test` to make them more OS-independent (very likely) and CI-friendly, then IMO we should, and then use them here, like we do in the current `.travis.yml`.

Reply:
(I realize `scripts/install` isn't currently used in `.travis.yml`, but I think we can very much make that work.)

Reply:
yes, it's convenient to have it one place, but scripts have one disadvantage is they concatenate logs from all steps, which is less convenient for failures in the middle.
And there can be quite a bit of them, `mypy`, `flake8`, `autoflake`, `black`, `coverage`, `bandit(?)` etc.
Per step granularity allows seeing the status and logs within the step.

Reply:
it's also what makes `tox` convenient, as keeps steps the same locally and in CI.
tho github actions have a local runner - https://github.com/nektos/act